### ISSUE TITLE
`Tools/unittestgui`: what should we do with it?

### ISSUE THREAD

**Issue by sobolevn** [python/cpython#111134]
I've asked @zware and @serhiy-storchaka whether this tool works.

They reported that there are problems with it.
![image](https://github.com/python/cpython/assets/4660275/1c4bac3a-ecb8-4e68-aa8c-4e56f3cbc848)

So, there are several options at the moment:
1. @zware proposed to move it to `unittest.gui`, if so we need to: start a proper discussion on discourse, fix all bugs, implement this feature / add tests / docs
2. We can just fix bugs and keep it as is, maybe mention in devguide
3. We can remove it, if other people don't find it useful anymore
4. We can just keep it as-is for history
5. Anything else? :)

**Comment by zware** [comment]
> They reported that there are problems with it.

The problems are not with the tool, but with how our test suite is set up (both Serhiy and I tried it on our tests, but it's a general tool).

If restricted to just one test package like `Lib/test/test_unittest/`, the tool works as intended as far as I can tell.  It is quite basic, though.

**Comment by serhiy-storchaka** [comment]
I do not think that we should do something with it right now. It is not broken, it may become more useful if the unittest CLI become more useful (support parallel running and other features of `test.regrtest`). It could even be integrated in IDLE if IDLE supported at least some kind of multifile projects.

**Comment by StanFromIreland** [comment]
I'm going to close this for now, since as Serhiy said, it's not broken currently and it doesn't carry a particularly significant maintenance burden.

**Comment by terryjreedy** [comment]
When editing 'codefile' in IDLE, I would like the File menu to have an 'Open test' option that would find and open 'test_codefile'.  It would need some configuration options for the find part.  If that were done, the Run menu could have a 'Test' option that used the same test-find machinery.  Other things first, though.  EDIT: I would start with an 'Ideas' discussion.
